test/JDBC/input/babel_613.sql

```diff
@@ -60,8 +60,12 @@ select coalesce(a, b) from t1;
 go
 
 -- test Union All
-select a from t1 Union All
-select b from t1;
+select * from 
```

Comment:
are we planning to add more tests? from code changes it seem there should be good number of new tests

Reply:
Added some test cases